source/_integrations/zha.markdown

```diff
@@ -316,6 +316,19 @@ To add a new device:
 1. Reset your Zigbee devices according to the device instructions provided by the manufacturer (e.g., turn on/off lights up to 10 times, switches usually have a reset button/pin). It might take a few seconds for the devices to appear. You can click on **Show logs** for more verbose output.
 1. Once the device is found, it will appear on that page and will be automatically added to your devices. You can optionally change its name and add it to an area (you can change this later). You can search again to add another device, or you can go back to the list of added devices.
 
+### Best practices for avoiding pairing difficulties
```

Comment:
Is it correct that this should be under "Adding devices"? Is under troubleshooting not a better place for this?

Reply:
I think that this should be under the "Adding devices" as these are things you always need to consider when adding Zigbee devices, but feel free to move it if you think that it fits better there. You can always move it later once this is merged.

Reply:
I'll leave that decision to the codeowners

Reply:
I think it makes more sense to have it under "adding devices" section.
Be, should the numbering be updated? Both are  1?

Reply:
Those numberings are not for the new section in this PR but for the old section above it. Regardless, the numberings in that othering section look correct when looking at the whole file.